extension/src/plots/webview/contract.ts

```diff
@@ -2,9 +2,10 @@ import { VisualizationSpec } from 'react-vega'
 import { Color } from '../../experiments/model/status/colors'
 
 export const PlotSizeNumber = {
-  LARGE: 500,
-  REGULAR: 400,
-  SMALL: 300
+  LARGE: 1,
```

Comment:
[C] Just an observation that these large plots generally don't fit on a single screen if the sidebar is hidden: 

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/37993418/203679724-b1decc2d-3958-46a7-9b71-a8439617b398.png">

Could set a max-height?

Reply:
Another observation: D'n'D a plot across sections (multiview vs single) when plots are large is nearly impossible:


https://user-images.githubusercontent.com/37993418/203681208-47f65bc7-119f-4807-a9c7-1c3e25e308a7.mov

Reply:
Ideally, resizing plots vertically would solve that, but I'll set a max-height until then.

Reply:
`max-height` conflicts with `aspect-ratio`. So I changed the ratio to be `9 / 5` instead of `4 / 3`. Looks good on my screen, until we get to 4 elements per row with the sidebar (this isn't the default number per row though). IMO, it's fine if it doesn't alway
<img width="1633" alt="Screenshot 2022-11-24 at 10 00 38 AM" src="https://user-images.githubusercontent.com/3683420/203815145-1942add6-478c-45e1-9843-95a54b315fa8.png">
s look perfect (there will always be small screens), that is why we allow resizing.
<img width="1632" alt="Screenshot 2022-11-24 at 10 00 54 AM" src="https://user-images.githubusercontent.com/3683420/203815156-d920d44a-9f52-4fec-9eed-8fd74923b6b4.png">
<img width="1631" alt="Screenshot 2022-11-24 at 10 01 14 AM" src="https://user-images.githubusercontent.com/3683420/203815174-92456b4e-ebb9-47cd-83f1-46520a372df3.png">